admin/sql/updates/20210319-mbs-11466.sql

```diff
@@ -0,0 +1,14 @@
+\set ON_ERROR_STOP 1
+BEGIN;
+
+ALTER TABLE language
+    ALTER COLUMN frequency SET DATA TYPE SMALLINT,
+    ALTER COLUMN frequency SET NOT NULL,
+    ALTER COLUMN frequency SET DEFAULT 0;
```

Comment:
We only really need the `SET DATA TYPE` bit; the others won't be affected by it.